data/os/RedHat.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I think these values can be moved to ```data/common.yaml```.

Reply:
I separated those into the RedHat.yaml as I anticipated them being different for the Debian family. In the original PR I saw for ubuntu support, the author used the value, /opt/caddy, www-data, www-data. Those are the user and group for web apps and seem to be part of the standard os distribution for Debian families, preconfigure with values that differ from the values currently used to manage the caddy user/group. My first thought was to use them as is, and add a module parameter to not manage the user/group with default true for RedHat and false for Debian, but perhaps the better way would be simply use same user/groups for Debian as RedHat? Thoughts?

Reply:
Makes sense, now that you explained it. ;)